### ISSUE TITLE
Transactions/scheduling meetups

### ISSUE THREAD

**Issue by coffeeScripter** [ka05/tiger-books#3]
How are we going to handle "transaction" or "schedule a meet"?

**Comment by ka05** [comment]
This is exactly why i think we have to make it an actual "native" app. basically we will have it send a text message or email or call to the owner of the book. ( the owner of the book could have settings in their app for what contact methods to allow. ) 

Another Idea could be simply a messaging system within this application but that would not be as informative to the user selling books, since they may not always be checking their messages within the app.

**Comment by coffeeScripter** [comment]
Lets just let them exchange email addresses at first then we can allow automated email from the site that does something generic like "You have an interested buyer. Here is their contact info."
This would need to be in the T&C for the site but that we would need smart peoples to help us with. I will see if Andrew can help with this if you say you agree.

**Comment by ka05** [comment]
Yes, definitely. I like this idea. this way  the seller does not get bombarded with text messages

**Comment by coffeeScripter** [comment]
also we could add a cap to the number of messages sent per day to a single person by a single user.

**Comment by ka05** [comment]
Yes, but the cap should only apply to the user sending the messages ( not the one receiving ). Right?

**Comment by coffeeScripter** [comment]
yes
